common/persistence/src/test/java/app/coronawarn/server/common/persistence/service/common/DiagnosisKeyExpirationCheckerTest.java

```diff
@@ -45,7 +46,7 @@ void shouldComputeThatKeyIsExpired(DiagnosisKey key, ExpirationPolicy expiration
   }
 
   private static Stream<Arguments> notExpiredKeysDataset() {
-    LocalDateTime midnight = LocalDateTime.of(LocalDate.now(), LocalTime.MIDNIGHT);
+    LocalDateTime midnight = LocalDateTime.of(LocalDate.now(ZoneId.of("UTC")), LocalTime.MIDNIGHT);
```

Comment:
consider usage of public statics `ZoneOffset.UTC` instead of fetching UTC with string